bootstrap-salt.sh

```diff
@@ -2516,6 +2516,10 @@ install_debian_8_git_deps() {
         __apt_get_install_noinput git || return 1
     fi
 
+    if [ "$(python -c 'import zmq; print zmq.pyzmq_version()')" = "" ]; then
```

Comment:
How about `try: import zmq; exit(0); except ImportError: exit(1);` and look for the exit code?